TSC/hosted-project-governance.md

```diff
@@ -0,0 +1,29 @@
+# Hosted Project Governance
+Published: March 10, 2025
+
+## Definition
+The Bytecode Alliance Technical Steering Committee (TSC) [charter](https://github.com/bytecodealliance/governance/blob/main/TSC/charter.md) defines two types of Bytecode Alliance Projects -- "Hosted" and "Core", saying:
+> By default a project adopted by the BA is considered a Hosted Project. The BA board designates specific projects at Core Projects, requiring adherence to heightened standards around project governance, mission alignment, communication, and security. … Core Projects must meet all the requirements of Hosted Projects, but each project has exactly one status – Hosted or Core.
+
+Further, the TSC charter lists these requirements for Hosted Projects:
+* Alignment with BA mission and goals
+* Open governance
+* Correct licenses or ability to change license to correct one
+* Apache 2.0 or exception granted by the board
```

Comment:
It's any of the licenses listed here; maybe we just link to it: https://github.com/bytecodealliance/governance/blob/main/templates/projects/proposal-core-project.md#licensing-compatible-with-the-bytecode-alliance

Reply:
Thanks.  I've modified the bullet list to just cite the details on "correct licenses" in the core project proposal template, linking to that document.